contracts/libraries/IterableOrderedOrderSet.sol

```diff
@@ -166,15 +163,13 @@ library IterableOrderedOrderSet {
         view
         returns (bytes32)
     {
+        require(value != QUEUE_END, "Trying to get next of last element");
+        bytes32 nextElement = self.nextMap[value];
         require(
-            value != QUEUE_END,
+            nextElement != bytes32(0),
             "Trying to get next of non-existent element"
         );
-        require(
-            self.nextMap[value] != bytes32(0),
-            "Trying to get next of last element"
-        );
-        return self.nextMap[value];
+        return nextElement;
```

Comment:
These changes are not necessary for this PR, but I decided to include them since I was at it. Note that the behavior of the function does not change. There are two conceptual modifications:
1. The messages in the `requre`s are swapped, to reflect what they are actually checking.
2. Reduce the number of storage read from two to one by storing the result of reading the first time in a variable `nextElement`.